### PR TITLE
CLEAN UP CLEAN UP EVERYONE LOVES CLEANING UP!

### PR DESCRIPTION
**Pull request by toyhammered**

### CODE REVIEW

**Comment by NuckChorris** [comment]
I see you added `package-lock.json` — that's the NPM lockfile, while the repo already has `yarn.lock` (the yarn lockfile)

You should discuss which package manager to use, yarn or npm, and stick to it 😅

**Comment by wopian** [comment]
Yarn

**Comment by toyhammered** [comment]
> I see you added `package-lock.json` — that's the NPM lockfile, while the repo already has `yarn.lock` (the yarn lockfile)
> 
> You should discuss which package manager to use, yarn or npm, and stick to it 😅

That was my bad. I totally thought there was a `package-lock.json`. Yeah we should use yarn for sure.

File: tsconfig.json [hunk 1]
```diff
@@ -1,19 +1,31 @@
 {
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
-    "jsx": "react",
-    // "lib": ["dom", "esnext"],
+    "jsx": "react-jsx",
     "target": "ES2019",
-    "module": "CommonJS",
+    "module": "esnext",
```
Comment (wopian) [inline tsconfig.json:6]:
```suggestion
    "module": "ESNext",
```

File: tsconfig.json [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+      "esnext"
+    ],
+    "allowJs": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "isolatedModules": true
+  },
+  "include": [
+    "src"
```
Comment (wopian) [inline tsconfig.json:29]: `app` if we want to be consistent with `kitsu-web` and `kitsu-server` folder naming

**Comment by toyhammered** [comment]
Dont need this anymore.

**Pull request closed without merging**